src/main/java/eu/yals/configuration/BugsnagConfig.java

```diff
@@ -0,0 +1,67 @@
+package eu.yals.configuration;
+
+import com.bugsnag.Bugsnag;
+import com.bugsnag.BugsnagSpringConfiguration;
+import eu.yals.constants.App;
+import eu.yals.utils.git.MavenGitInfo;
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+import org.springframework.core.env.Environment;
+
+import java.net.InetSocketAddress;
+import java.net.Proxy;
+
+@Slf4j
+@Configuration
+@Import(BugsnagSpringConfiguration.class)
+public class BugsnagConfig {
+    private static final String TAG = "[BugsnagConfig]";
+    private static final String NO_TOKEN = "noToken";
+
+    private final Environment env;
+    private MavenGitInfo mavenGitInfo;
+
+    public BugsnagConfig(Environment env, MavenGitInfo gitInfo) {
+        this.env = env;
+        this.mavenGitInfo = gitInfo;
+    }
+
+    private String proxyHost;
+    private String proxyPort;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Fields should be declared at the top of the class, before any method declarations, constructors, initializers or inner classes.](https://app.codacy.com/gh/yadevee/yals/pullRequest?prid=5392475)